examples/solid/authenticated-routes-firebase/src/routes/login.tsx

```
@@ -0,0 +1,106 @@
+import {
+  createFileRoute,
+  redirect,
+  useRouter,
+  useRouterState,
```

ESLint

[error] 'useRouterState' is defined but never used.

(unused-imports/no-unused-imports)